File: Normal/0102/main.swift
public class TreeNode {
    public var val: Int
    public var left: TreeNode?
    public var right: TreeNode?
    public init() { self.val = 0; self.left = nil; self.right = nil; }
    public init(_ val: Int) { self.val = val; self.left = nil; self.right = nil; }
    public init(_ val: Int, _ left: TreeNode?, _ right: TreeNode?) {
        self.val = val
        self.left = left
        self.right = right
    }
}

class Solution {
    func levelOrder(_ root: TreeNode?) -> [[Int]] {
        guard let root = root else {
            return []
        }
        var array = [[Int]]()
        var row: [TreeNode] = [root]
        var nodeArray = [TreeNode]()
        var valueArray = [Int]()
        while row.count != 0{
            nodeArray = []
            valueArray = []
            for node in row{
                valueArray.append(node.val)
                if let left = node.left{
                    nodeArray.append(left)
                }
                if let right = node.right{
                    nodeArray.append(right)
                }
            }
            array.append(valueArray)
            row = nodeArray
        }
        return array
    }
}

File: Normal/0102/main.py
from typing import List


class TreeNode:
    def __init__(self, val=0, left=None, right=None):
        self.val = val
        self.left = left
        self.right = right
        
class Solution:
    def levelOrder(self, root: TreeNode) -> List[List[int]]:
        if root == None:
            return []
        array: List[List[int]] = []
        row: List[TreeNode] = [root]
        nodeArray: List[TreeNode] = []
        valueArray: List[int] = []
        node: TreeNode = None
        while len(row) != 0:
            nodeArray = []
            valueArray = []
            for node in row:
                valueArray.append(node.val)
                if node.left != None:
                    nodeArray.append(node.left)
                if node.right != None:
                    nodeArray.append(node.right)
            row = nodeArray
            array.append(valueArray)
        return array